tests/test_collapse.py

```diff
@@ -38,9 +38,9 @@ def collapse_adjacent_sequences(tree: ete3.TreeNode) -> ete3.TreeNode:
 etetree2 = utils.collapse_adjacent_sequences(etetree.copy())
 
 
-with open("sample_data/toy_trees_100_collapsed.p", "rb") as fh:
+with open("../sample_data/toy_trees_100_collapsed.p", "rb") as fh:
```

Comment:
It looks like you must be running `pytest` from within the `tests/` directory. Instead it's standard to run it from the root project directory, so these file paths should be changed back to get the GitHub CI tests to run